tools/data_loader.py

```diff
@@ -197,7 +197,18 @@ def inject_references(obj):
                 elif filename.endswith('reg'):
                     return Regions.read(filename).serialize(format='ds9')
                 elif filename.endswith('h5') or filename.endswith('hdf5'):
-                    payload = Table.read(filename).to_pandas().to_dict(orient='list')
+                    try:
+                        payload = (
+                            Table.read(filename).to_pandas().to_dict(orient='list')
+                        )
+                    except Exception as e:
+                        # sometimes we save HDF5 files using an HDFStore.
+                        # in this case we read it as a binary file and return it as "data"
+                        if 'values_block_0' in str(e):
+                            with open(filename, 'rb') as fid:
+                                payload = base64.b64encode(fid.read())
```

Comment:
Should this also have its own try: except?

Reply:
Not really. All the exceptions are caught at the end and then there's an error log. 
I put an exception clause here just to be able to fish out the specific h5 files that I was making for the photometric series (although anyone using the `pd.HDFStore` method will be able to use this). 
If anything goes wrong reading that file it would just put an `X` there and report the error at the end.

Reply:
Ok!